client/src/components/Export/ExportToZipButton/ExportToZipButton.js

```diff
@@ -24,20 +24,30 @@ export const ExportToZipButton = (props) => {
         zip.file(exportFileName, csv);
         if (images){
             for (const image of images) {
-                await JSZipUtils.getBinaryContent(image.url, {}).then(
-                    (data, err) => {
-                        if (err) {
-                            alert('Problem happened when download img: ' + image.url);
-                            console.error('Problem happened when download img: ' + image.url);
-                        } else {
-                            let name = getS3ImageNameFromObjectUrl(image.url)
-                            if (!name.endsWith('.png') && !name.endsWith('.jpg')) {
-                                name = name + '.jpg'
-                            }
-                            zip.file(name, data, {binary: true});
-                        }
+                // Setup the global AWS config
+                AWS.config.update({
+                    region: 'us-east-2',
+                    accessKeyId: `${process.env.REACT_APP_AWS_ACCESS_KEY_ID}`,
+                    secretAccessKey: `${process.env.REACT_APP_AWS_SECRET_ACCESS_KEY}`
+                });
+                const key = getS3ImageObjectKeyFromObjectUrl(image.url)
+                const s3Client = new AWS.S3();
+
+                try {
+                    let data = await s3Client.getObject({
+                        Bucket: 'monuments-and-memorials',
+                        Key: key
+                    }).promise();
+                    let name = getS3ImageNameFromObjectUrl(image.url)
+                    if (!name.endsWith('.png') && !name.endsWith('.jpg')) {
+                        name = name + '.jpg'
                     }
-                )
+                    zip.file(name, data.Body, {binary: true});
+                    console.log(data)
+                } catch (e) {
```

Comment:
Do we want to do anything with the caught exception?

Reply:
added rollbar logging